coalib/bears/requirements/PackageRequirement.py

```diff
@@ -49,3 +49,38 @@ def check(self):
         :return: Returns True if satisfied, False if not.
         """
         raise NotImplementedError
+
+    @classmethod
+    def multiple(cls, *args):
+        """
+        Creates a tuple of multiple instances of a class.
+
+        Should not be instances of ``PackageRequirement``, as this is an
+        abstract class:
+
+        >>> PackageRequirement.multiple(('pip', 'coala_decorators', '0.1.0'),)
+        Traceback (most recent call last):
+        ...
+        NotImplementedError
```

Comment:
you would rather make an example here of a subclass and how it has to be implemented so this works